diff --git a/CMakeLists.txt b/CMakeLists.txt
index 473b737..f18d415 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -127,14 +127,14 @@ if (Qt5Widgets_FOUND AND Qt5Test_FOUND)
   
   add_executable(notebook ${gui_main} ${gui_src})
   if(UNIX AND NOT APPLE AND CMAKE_COMPILER_IS_GNUCXX)
-    target_link_libraries(notebook interpreter Qt5::Widgets pthread)
+    target_link_libraries(notebook interpreter Qt5::Widgets pthread gcov)
   else(UNIX AND NOT APPLE AND CMAKE_COMPILER_IS_GNUCXX)
     target_link_libraries(notebook interpreter Qt5::Widgets)
   endif()
 
   add_executable(notebook_test ${gui_test_src} ${gui_src})
   if(UNIX AND NOT APPLE AND CMAKE_COMPILER_IS_GNUCXX)
-    target_link_libraries(notebook_test interpreter Qt5::Widgets Qt5::Test pthread)
+    target_link_libraries(notebook_test interpreter Qt5::Widgets Qt5::Test pthread gcov)
   else(UNIX AND NOT APPLE AND CMAKE_COMPILER_IS_GNUCXX)
     target_link_libraries(notebook_test interpreter Qt5::Widgets Qt5::Test)
   endif()
